### PR TITLE
Issue clycities frontend#16

### LINKED ISSUE #16: [First Timers Only] Create a README for ClyCites Frontend
# 🐥 First Timers Only – Good First Issue

This issue is reserved for **new contributors** to this repository or anyone who is **new to open source** in general.  
We understand that making your **first pull request (PR)** can be overwhelming, and we're here to help!

The purpose of this issue—and others labeled with `good first issue`—is to guide you through your **first open-source contribution**.

---

## 👾 Issue Description

The **ClyCites Frontend** repository currently does not have a README file, which makes it hard for new contributors to understand the project setup, structure, and contribution guidelines.

Your task is to **create a comprehensive README file** that includes the following sections:

✅ **Project Name and Description**  
(Briefly describe the purpose of the ClyCites Frontend app)

✅ **Features**  
(List major features like Dashboard, GitHub Issues Display, etc.)

✅ **Tech Stack**  
(Example: Next.js, TypeScript, Tailwind CSS, shadcn/ui)

✅ **Getting Started**  
(Steps to set up the project locally, including installation and running instructions)

✅ **Environment Variables**  
(Mention required variables and how to configure them)

✅ **Contributing Guidelines**  
(Basic contribution guide and reference to CODE_OF_CONDUCT.md if available)

✅ **License**  
(Specify the project license, e.g., MIT)

## ✅ Suggested Solution

Create or extend the appropriate test class (e.g., `{{ClassName}}Test`) to add full coverage for the `{{methodName}}` method.

Follow existing testing patterns (likely using **JUnit** or the relevant test framework).  
Ensure to include **positive and negative** test cases.

---

## 📋 Step-by-Step Guide to Contribute

1. **Comment below** to claim this issue.

2. Wait for a **maintainer to assign** it to you.

3. **Fork** the repo, then **clone** your fork locally.

4. Create a new branch:

  ```bash
    git checkout -b issue-{{issueNumber}}-add-tests
  ```

### 🛠️ Implement the Fix

- Add or update the necessary tests in the codebase.

- Ensure your tests cover all edge cases and scenarios mentioned in the issue description.

- Commit your changes with a clear message:

  ```bash
    git commit -s -m "test: add coverage for {{methodName}} in {{ClassName}}"
  ```
- Ensure your commit is signed off with `-s` to comply with the Developer Certificate of Origin.

- Push your branch to your forked repository:

  ```bash
    git push origin issue-{{issueNumber}}-add-tests
  ```

- Open a **Pull Request (PR)** to the main repository.

- Monitor **GitHub Actions** – if any checks fail and you're unsure why, feel free to ask for help in the PR comments.

- Respond to feedback and make any necessary changes requested during the code review.

- Once approved, your PR will be merged by a maintainer.

- Celebrate your first contribution to open source! 🎉

### PR DESCRIPTION
**Pull request by Rohiitt405**
#### Summary of Changes (What does this PR do?)

This PR adds a comprehensive README.md file to the project repository. It addresses the lack of project documentation, which made it difficult for new contributors to understand the project's purpose, setup, and contribution process.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [x] I've included issue number in the "Closes #16" part of the section to [link the issue](https://github.com/ClyCites/ClyCites-Frontend/issues/16).
- [x] I've updated corresponding documentation for the changes in this PR.
- [x] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
1. View the README.md file in the "Files changed" tab of this PR.
2. Verify that the file renders correctly with proper Markdown formatting.
3. Read through each section to ensure the information is clear, accurate, and covers all the required topics.
Optionally, follow the Getting Started instructions to confirm they are correct for a new contributor.

#### What are the relevant tickets?

- Closes #16 

#### Screenshots (optional)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,303 @@
...
+## 🙏 Acknowledgments
+
+- **Management Team** - For vision and leadership
+- **All Contributors** - For their valuable contributions
+- **Agricultural Experts** - For domain knowledge and insights
+- **Open Source Community** - For the amazing tools and libraries
+
+## 📞 Contact
+
+- **Email**: info@open-elements.com
```
Comment (danielmarv) [inline README.md]: Use clycites@gmail.com for the email contact

**Review by danielmarv — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,74 +1,303 @@
...
+
+- **Management Team** - For vision and leadership
+- **All Contributors** - For their valuable contributions
+- **Agricultural Experts** - For domain knowledge and insights
+- **Open Source Community** - For the amazing tools and libraries
+
+## 📞 Contact
+
+- **Email**: clycites@gmail.com
+- **GitHub Issues**: [Create an issue](https://github.com/Saga211/ClyCites-Frontend/issues)
```
Comment (danielmarv) [inline README.md]: Why r u use using Saga211 account this is and org

**Review by danielmarv — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Rohiitt405) [reply README.md]: should i also make changes on github discussion link ?
Screenshot 2025-08-01 002402

File: README.md [hunk 3]
```diff
@@ -1,74 +1,303 @@
...
+
+- **Management Team** - For vision and leadership
+- **All Contributors** - For their valuable contributions
+- **Agricultural Experts** - For domain knowledge and insights
+- **Open Source Community** - For the amazing tools and libraries
+
+## 📞 Contact
+
+- **Email**: clycites@gmail.com
+- **GitHub Issues**: [Create an issue](https://github.com/ClyCites/issues)
```
Comment (danielmarv) [inline README.md]:
```suggestion
- **GitHub Issues**: [Create an issue](https://github.com/ClyCites/ClyCites-Frontend/issues)
```

File: README.md [hunk 4]
```diff
@@ -1,74 +1,303 @@
...
+- **Discussions**: [Join the discussion](https://github.com/ClyCites/discussions)
```
Comment (danielmarv) [inline README.md]:
```suggestion
- **Discussions**: [Join the discussion](https://github.com/ClyCites/ClyCites-Frontend/discussions)
```
